### ISSUE TITLE
[Question] upstream server Priority order

### ISSUE THREAD

**Issue by ChaosNicro** [IngoZenz/personaldnsfilter#258]
Heya,
I'm coming here from DNS66 since that is no longer receiving updates. I really like the raw access approach while still keeping a convenient and helpful UI. Makes for a lot of fine user control.

I tried adding ffmuc.net as a local DoT provider above the defaults and had it consistently skip the entries in favor of the UDP or DoH defaults. I was going to report a bug but after removing the defaults it's now taking the custom entries.
I was naively assuming that the first reachable server in the list would be consistently chosen on startup but it might go through them at random. Could you clear that up?
If that is not expected behaviour I'd be happy to provide my configs for testing.

Thank you for providing this project.

**Comment by ChaosNicro** [comment]
Also it seems like additional comments and empty lines are stripped on save. They would make organizing and readability easier.

**Comment by ChaosNicro** [comment]
I noticed from other issues that itis meant to select the fastest by speed. A bit odd that a server local to my country is slower than the defaults, but ok. Assuming that is the whole selection strategy, my question is answered. Would be good to add that info into the help or the config TXT if it isn't already and I just missed it.

**Comment by IngoZenz** [comment]
Yes - actually it takes the fastest. You can see the measured speed within dnsperf.info in the app directory.
The dns config UI will be improved with next release.
